core/app/models/spree/stock/estimator.rb

```diff
@@ -20,20 +22,17 @@ def shipping_rates(package, frontend_only = true)
         rates = calculate_shipping_rates(package)
         rates.select! { |rate| rate.shipping_method.frontend? } if frontend_only
         choose_default_shipping_rate(rates)
-        sort_shipping_rates(rates)
+        shipping_rate_sorter_class.new(rates).sort
```

Comment:
Also there is no spec that verifies the `shipping_rate_sorter_class` can be configured externally. This code could just hardcode the default class and no spec would notice.